layouts/error.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <section class="container">
+    <img src="../assets/img/logo.png" alt="Nuxt.js Logo" class="logo" />
```

Comment:
`../assets/img/logo.png` is missing

Reply:
It is not missing on my setting. The file is checking here: https://github.com/rickmak/puttyimages-web/blob/0d6ee56c849ddbe65795099d5abbea773f8359b0/assets/img/logo.png


Just to confirm the path resolve setting is right. I will remove the dummy img.